packages/aws-cdk-lib/aws-dynamodb/test/table-v2.test.ts

```diff
@@ -1293,6 +1293,78 @@ describe('replica tables', () => {
     });
   });
 
+  test('with TagAspect', () => {
+    // GIVEN
+    const stack = new Stack(undefined, 'Stack', { env: { region: 'us-east-1' } });
+
+    // WHEN
+    const table = new TableV2(stack, 'Table', {
+      partitionKey: { name: 'pk', type: AttributeType.STRING },
+      replicas: [{
+        region: 'us-west-1',
+      }],
+    });
+
+    Tags.of(table).add('tagKey', 'tagValue');
```

Comment:
can you add a new test case where you can Tag a parent constructor like the stack itself, and figure out if the Table resource will be tagged as with. Like for example I want to tag all resources in Production stack as `resourceType - > Prod`

Reply:
also add it to the integ testing